go/libraries/doltcore/sqle/dsess/transactions.go

```diff
@@ -36,6 +37,8 @@ const (
 	maxTxCommitRetries = 5
 )
 
+var ErrRetryTransaction = errors.New("another client has modified the working set incompatibly. retry the transaction")
```

Comment:
maybe something more like:
"this transaction conflicts with a committed transaction from another client, please retry"